src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.php

```diff
@@ -63,7 +63,7 @@
         ->set('validator.mapping.cache.adapter', PhpArrayAdapter::class)
             ->factory([PhpArrayAdapter::class, 'create'])
             ->args([
-                param('validator.mapping.cache.file'),
+                param('kernel.build_dir').'/'.param('validator.mapping.cache.file'),
```

Comment:
this changes the meaning of the parameter, potential BC break ahead

Reply:
Reverted to the previous meaning